<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrimerDispListDlg</class>
 <widget class="QDialog" name="PrimerDispListDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Display Primer List</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPlainTextEdit" name="plaintext"/>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_all">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Display all primer pairs</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>OR</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QCheckBox" name="checkBox_topPrimers">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Display top</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_topNum">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>500</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>primer pairs</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>OR</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Display partial primer pair list according to</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Condition 1</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="20,1,0,0">
           <item>
            <widget class="QComboBox" name="comboBox_cond1"/>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_GTLT1">
             <item>
              <property name="text">
               <string>&lt;</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>&gt;</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_value1">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_percent1">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Condition 2</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="20,1,0,0">
           <item>
            <widget class="QComboBox" name="comboBox_cond2"/>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_GTLT2">
             <item>
              <property name="text">
               <string>&lt;</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>&gt;</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_value2">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_percent2">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Condition 3</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="20,1,0,0">
           <item>
            <widget class="QComboBox" name="comboBox_cond3"/>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_GTLT3">
             <item>
              <property name="text">
               <string>&lt;</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>&gt;</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_value3">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_percent3">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Condition 4</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="20,1,0,0">
           <item>
            <widget class="QComboBox" name="comboBox_cond4"/>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_GTLT4">
             <item>
              <property name="text">
               <string>&lt;</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>&gt;</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_value4">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_percent4">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>AND</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>AND</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>AND</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PrimerDispListDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>640</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PrimerDispListDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>640</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_cond1</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PrimerDispListDlg</receiver>
   <slot>changeIdx_1(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_cond2</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PrimerDispListDlg</receiver>
   <slot>changeIdx_2(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_cond3</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PrimerDispListDlg</receiver>
   <slot>changeIdx_3(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>476</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_cond4</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PrimerDispListDlg</receiver>
   <slot>changeIdx_4(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>572</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_topPrimers</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PrimerDispListDlg</receiver>
   <slot>displayTop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_all</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PrimerDispListDlg</receiver>
   <slot>displayAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeIdx_1(int)</slot>
  <slot>changeIdx_2(int)</slot>
  <slot>changeIdx_3(int)</slot>
  <slot>changeIdx_4(int)</slot>
  <slot>displayTop()</slot>
  <slot>displayAll()</slot>
 </slots>
</ui>
